src/backend/distributed/sql/citus--11.0-2--11.0-3.sql

```diff
@@ -0,0 +1 @@
+#include "udfs/citus_finalize_upgrade_to_citus11/11.0-3.sql"
```

Comment:
do we need to change version to 11.0-3 in any other place?

Reply:
Yes, this isn't used without a control file. We also need to rewire the 11.1-1 script and add appropriate downgrade scripts.

Reply:
One idea @hanefi brought up, which I also like, is to squeeze 11.0-3 in between 11.0-2 -> 11.1-1, given that we have NOT released 11.1 yet.

So, make 11.0-3 as if we are already developing 11.0 release.

Does that sound OK to you as well?

Reply:
sure, I cannot think of any other thing to do 

we also need to update the multi_extension tests probably.

Reply:
@hanefi do you have any idea on this?

Reply:
I am not so sure as well. However, your changes look reasonable to me.

Reply:
Thanks, still waiting @marcocitus to approve regarding the script changes. Manually tested, but non-trivial to add a regression test.

I'll try to add that with a follow up PR, because this is something to keep in mind for future releases as well.

Reply:
Looks fine, the backport will require some changes (control file, no 11.1 files)

Reply:
After this PR is merged, Let me do a review for the backport just to be safe.

Reply:
https://github.com/citusdata/citus/pull/6037/files for reference